Initial

Aiksterzh opened the pull request:
- initial file setup
- fix endpoints
- set up postgres schema

Aiksterzh committed:
Change endpoints to display appropriate data for each listing

Aiksterzh committed:
Update schema to use appropriate databse

Aiksterzh committed:
Add postgress database
- write new schema

vlad-pisotskyi commented on database/postgress.sql:
```diff
@@ -0,0 +1,65 @@
+DROP TABLE IF EXISTS users;
+
+CREATE TABLE users (
+  id BIGINT NOT NULL,
```
I think it's BIGSERIAL

vlad-pisotskyi commented on database/postgress.sql:
```diff
@@ -0,0 +1,65 @@
+DROP TABLE IF EXISTS owners;
+
+CREATE TABLE owners (
+  id BIGINT NOT NULL,
```
BIGSERIAL

vlad-pisotskyi commented on .gitignore:
```diff
@@ -0,0 +1,25 @@
+
+# Compiled Dirs (http://nodejs.org/api/addons.html)
+build/
+dist/
```
Newline required at end of file

vlad-pisotskyi commented on client/src/components/Booking.jsx:
```diff
@@ -145,7 +145,8 @@ class Booking extends Component {
 
   checkoutWithTripDetails(tripDetails) {
-    axios.post('/booking', tripDetails)
+    let id = window.location.pathname;
```
you can put this in props, not to repeat yourself.
(i know its STC not FEC) XD

Pisotski reviewed (commented):
I am not sure about BIGSERIAL (I didn't check myself).
Looks good

Aiksterzh committed:
Implement changes:
- set id as props
- change schema to autoincrement values
- add spacing to gitignore

Aiksterzh committed:
Refactor code

Pisotski reviewed (approved):
looks good
